package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "chartjs-plugin-zoom",
   "description": "Simple HTML5 charts using the canvas element.",
```

Comment:
Should we update the package description?

Reply:
done